e2e-tests/playwright/lib/src/server/playwright_client.ts

```
@@ -47,4 +51,32 @@ export class PlaywrightClient4 extends Client4 {
+        if (!post.file_ids) {
+            post.file_ids = await Promise.all(
+                files.map((filename) => {
+                    return new Promise<string>(async (resolve) => {
+                        const formData = new FormData();
+                        formData.set('channel_id', post.channel_id);
+                        formData.set('files', getFileFromAsset(filename), filename);
+
+                        const data = await this.uploadFile(formData);
+
+                        resolve(data.file_infos[0].id);
+                    });
```

Biome (2.5.3)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)